services/accounts/settings.go

```diff
@@ -41,3 +41,118 @@ func (api *SettingsAPI) NodeConfig(ctx context.Context) (*params.NodeConfig, err
 func (api *SettingsAPI) SaveNodeConfig(ctx context.Context, n *params.NodeConfig) error {
 	return nodecfg.SaveNodeConfig(api.db.DB(), n)
 }
+
+// Notifications Settings
+func (api *SettingsAPI) GetAllowNotifications() (bool, error) {
+	return api.db.GetAllowNotifications()
+}
+
+func (api *SettingsAPI) SetAllowNotifications(value bool) error {
+	return api.db.SetAllowNotifications(value)
+}
+
+func (api *SettingsAPI) GetOneToOneChats() (string, error) {
```

Comment:
This function will be available as `settings_getOneToOneChats`. Without knowing the context or implementation, it's difficult to know that this function deals with notifications. Consider either adding a prefix like "Notif" to the function name, or using a different service.  (same observation applies to other functions)